doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -188,6 +169,38 @@ preamble similar to the one below, indicating that this is an executable file:
 
 You don't need to edit this preamble, as it is autogenerated.
```

Comment:
Clearly this does not need to have a full markdown tutorial, but there are a few points I think might be worth calling out here:
- Best practices for adding images. (E.g. where should the files go?)
- Best practices for adding links to RST documentation from notebook tutorials. Should I just add a regular hyperlink [like this](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.gmean.html#scipy.stats.gmean) or is there a fancy way like RST references / intersphinx links?
- How to link to notebook tutorials from RST documentation.

Reply:
Let me know if this is reasonable. If you want to go through these instructions to translate a notebook before merging, that could be a good way to catch anything missing from the instructions.